src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -17,15 +17,12 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Avoid wildcard imports. (You can and should change your IDE's settings, which are probably what triggered this change - I have IntelliJ set to automatically insert a wildcard import only if there are more than 10,000 imports from a single package, which never occurs in practce.)